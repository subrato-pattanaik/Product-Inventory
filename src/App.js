import React from "react";
import AllRoute from "./components/AllRoute/AllRoute";
import { BrowserRouter as Router } from "react-router-dom";

function App() {
  return (
    <Router>
      <AllRoute />
    </Router>
  );
}

export default App;
