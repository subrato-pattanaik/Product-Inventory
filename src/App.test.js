import { shallow } from "enzyme";
import React from "react";
import App from "./App";

it("should test snapshot of App page", () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
it("should test All Route component exist or not", () => {
  const wrapper = shallow(<App />);
  expect(wrapper.find("AllRoute").exists()).toEqual(true);
});
