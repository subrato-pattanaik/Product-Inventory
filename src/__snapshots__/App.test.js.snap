// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test snapshot of App page 1`] = `
<BrowserRouter>
  <AllRoute />
</BrowserRouter>
`;
